src/hooks/useOpenInBrowser.ts

```diff
@@ -0,0 +1,23 @@
+import { Navigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import { Linking } from 'react-native';
+
+export function useOpenInBrowser() {
+  return async (url: string, internal = true) => {
+    const isDeeplink = !url.startsWith('http');
+    if (isDeeplink) {
+      const supported = await Linking.canOpenURL(url);
+      if (supported) {
+        return Linking.openURL(url);
+      }
+      console.log('No deeplinking available for url:', url);
```

Comment:
Sorry, I think this was definitely my fault, but let's change this to logger.warn so we can capture it in sentry.